src/data/trace.ts

```diff
@@ -14,7 +14,7 @@ interface BaseTraceStep {
 }
 
 export interface TriggerTraceStep extends BaseTraceStep {
-  changed_variables: {
+  changed_variables?: {
```

Comment:
This is mainly needed to fix the gallery pages.

Reply:
Can't we just add an empty `changed_variables` to the mocks in gallery?